crates/base/src/global/veci8.rs

```diff
@@ -0,0 +1,184 @@
+use crate::{
+    global::Veci8Owned,
+    scalar::{F32, I8},
+};
+
+use super::Veci8Borrowed;
+
+pub fn dot(x: &[I8], y: &[I8]) -> F32 {
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
`x86` refers to `i386`, `i586` and `i686` in Rust.

Also, we do not support any 32-bit targets.